src/masternodes/govvariables/attributes.cpp

```diff
@@ -948,6 +1073,29 @@ Res ATTRIBUTES::Validate(const CCustomCSView & view) const
                 }
             break;
 
+            case AttributeTypes::Consortium:
+                switch (attrV0->key) {
+                    case ConsortiumKeys::Members:
+                        if (view.GetLastHeight() < Params().GetConsensus().GreatWorldHeight) {
```

Comment:
GreatWorldHeight -> GrandCentralHeight ?

Reply:
That's weird, it already is GrandCentral.

https://github.com/DeFiCh/ain/blob/featuer/consortium/src/masternodes/govvariables/attributes.cpp#L1194

Reply:
I was in a different commit when I did this review. Resolving conversation.